data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -275,17 +273,12 @@ private CloseableIterable<Record> openDeletes(DeleteFile deleteFile, Schema dele
     InputFile input = getInputFile(deleteFile.path().toString());
     switch (deleteFile.format()) {
       case AVRO:
-        return Avro.read(input)
-            .project(deleteSchema)
-            .reuseContainers()
-            .createReaderFunc(DataReader::create)
-            .build();
+        return Avro.read(input).project(deleteSchema).createReaderFunc(DataReader::create).build();
```

Comment:
It should fail the code style checking.

Reply:
That's kind of what I thought, too. Spotless made this change, and I wasn't sure if I should fight against the embedded code formatter rules.

Reply:
PR checks fail after wrapping :(

```
> The following files had format violations:
See https://docs.gradle.org/8.1.1/userguide/command_line_interface.html#sec:command_line_warnings
      src/main/java/org/apache/iceberg/data/DeleteFilter.java
62 actionable tasks: 62 executed
          @@ -273,10 +273,7 @@
           ····InputFile·input·=·getInputFile(deleteFile.path().toString());
           ····switch·(deleteFile.format())·{
           ······case·AVRO:
          -········return·Avro.read(input)
          -············.project(deleteSchema)
          -············.createReaderFunc(DataReader::create)
          -············.build();
          +········return·Avro.read(input).project(deleteSchema).createReaderFunc(DataReader::create).build();
           
           ······case·PARQUET:
           ········Parquet.ReadBuilder·builder·=
  Run './gradlew :iceberg-data:spotlessApply' to fix these violations.
```